src/languages/en.js

```diff
@@ -854,6 +854,7 @@ export default {
         growlMessageEmptyName: 'Please provide both a first and last name so our guides know how to address you!',
         callButton: 'Call',
         callButtonTooltip: 'Get live help from our team',
+        blockedFromConcierge: 'Due to previous interactions with our staff, call cannot be scheduled at this time.',
```

Comment:
Copy: should this be `calls cannot be scheduled..`?

Reply:
Maybe "a call" is better. I'll update.